org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -41,6 +41,8 @@
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/529">#529</a>).</li>
   <li>Maven aggregated reports will now also include modules of runtime dependencies
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/498">#498</a>).</li>
+  <li>JaCoCo now comes with a simple command line interface
```

Comment:
@Godin I want to reword/re-arrange the whole block anyways before we do a 0.8.0 release. Can we do this in a separate task for 0.8.0?